atomic_reactor/plugins/pre_pull_base_image.py

```diff
@@ -19,33 +19,30 @@
 import docker
 
 from atomic_reactor.plugin import PreBuildPlugin
-from atomic_reactor.util import (get_build_json, get_manifest_list,
-                                 get_config_from_registry,
-                                 get_platforms, base_image_is_custom,
+from atomic_reactor.util import (get_build_json, get_platforms, base_image_is_custom,
                                  get_checksums, get_manifest_media_type,
-                                 get_all_manifests)
+                                 RegistrySession, RegistryClient)
 from atomic_reactor.core import RetryGeneratorException
 from atomic_reactor.plugins.pre_reactor_config import (get_source_registry,
                                                        get_platform_to_goarch_mapping,
-                                                       get_registries_organization)
+                                                       get_registries_organization,
+                                                       get_pull_registries)
 from io import BytesIO
 from requests.exceptions import HTTPError, RetryError, Timeout
-from osbs.utils import RegistryURI, ImageName
+from osbs.utils import ImageName
 
 
 class PullBaseImagePlugin(PreBuildPlugin):
     key = "pull_base_image"
     is_allowed_to_fail = False
 
-    def __init__(self, tasker, workflow, parent_registry=None, parent_registry_insecure=False,
-                 check_platforms=False, inspect_only=False, parent_images_digests=None):
+    def __init__(self, tasker, workflow, check_platforms=False, inspect_only=False,
+                 parent_images_digests=None):
         """
         constructor
 
         :param tasker: ContainerTasker instance
         :param workflow: DockerBuildWorkflow instance
-        :param parent_registry: registry to enforce pulling from
-        :param parent_registry_insecure: allow connecting to the registry over plain http
```

Comment:
I was trying to find where these arguments were passed from and found them in `api.py`, is that OK? Do we want to leave them there?

Reply:
I've deleted em from api as well